### PR TITLE
Add input boxes for required user inputs

### PR DESCRIPTION
**Pull request by claudevdm**
Add input boxes to user required inputs in example notebooks.

Colab diffs:
[automatic_model_refresh](https://colab.research.google.com/diff#left=https%3A%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fautomatic_model_refresh.ipynb&right=https%3A%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fautomatic_model_refresh.ipynb)

[automatic_model_refresh.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fautomatic_model_refresh.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fautomatic_model_refresh.ipynb)
[bigquery_enrichment_transform.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fbigquery_enrichment_transform.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fbigquery_enrichment_transform.ipynb)
[bigtable_enrichment_transform.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fbigtable_enrichment_transform.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fbigtable_enrichment_transform.ipynb)
[vertex_ai_text_embeddings.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fdata_preprocessing%2Fvertex_ai_text_embeddings.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fdata_preprocessing%2Fvertex_ai_text_embeddings.ipynb)
[dataframe_api_preprocessing.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fdataframe_api_preprocessing.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fdataframe_api_preprocessing.ipynb)
[gemma_2_sentiment_and_summarization.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fgemma_2_sentiment_and_summarization.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fgemma_2_sentiment_and_summarization.ipynb)
[nlp_tensorflow_streaming.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fnlp_tensorflow_streaming.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fnlp_tensorflow_streaming.ipynb)
[run_inference_pytorch.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_pytorch.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_pytorch.ipynb)
[run_inference_sklearn.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_sklearn.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_sklearn.ipynb)
[run_inference_tensorflow.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_tensorflow.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_tensorflow.ipynb)
[run_inference_tensorflow_with_tfx.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_tensorflow_with_tfx.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_tensorflow_with_tfx.ipynb)
[run_inference_vertex_ai.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_vertex_ai.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_vertex_ai.ipynb)
[run_inference_vllm.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_vllm.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Frun_inference_vllm.ipynb)
[vertex_ai_feature_store_enrichment.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fvertex_ai_feature_store_enrichment.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fbeam-ml%2Fvertex_ai_feature_store_enrichment.ipynb)
[beam_nlp.ipynb](https://colab.research.google.com/diff#left=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fmaster%2Fexamples%2Fnotebooks%2Fhealthcare%2Fbeam_nlp.ipynb&right=https:%2F%2Fgithub.com%2Fapache%2Fbeam%2Fblob%2Fe6c456ec2309e34eaee9b1f140296fa011995350%2Fexamples%2Fnotebooks%2Fhealthcare%2Fbeam_nlp.ipynb)
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by claudevdm** [comment]
R: @damccorm

File: examples/notebooks/beam-ml/automatic_model_refresh.ipynb [hunk 1]
```diff
@@ -244,135 +233,145 @@
...
+          "text": [
+            "\n"
+          ]
+        }
+      ],
       "source": [
         "model = tf.keras.applications.resnet.ResNet101()\n",
         "model.save('resnet101_weights_tf_dim_ordering_tf_kernels.keras')\n",
         "# After saving the model locally, upload the model to GCS bucket and provide that gcs bucket `URI` as `model_uri` to the `TFModelHandler`\n",
         "# Replace `BUCKET_NAME` value with actual bucket name.\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/automatic_model_refresh.ipynb:301]: Can we get rid of this comment now?

File: examples/notebooks/beam-ml/automatic_model_refresh.ipynb [hunk 2]
```diff
@@ -534,108 +541,118 @@
...
+          "output_type": "stream",
+          "text": [
+            "\n"
+          ]
+        }
+      ],
       "source": [
         "model = tf.keras.applications.resnet.ResNet152()\n",
         "model.save('resnet152_weights_tf_dim_ordering_tf_kernels.keras')\n",
         "# Replace the `BUCKET_NAME` with the actual bucket name.\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/automatic_model_refresh.ipynb:607]: Same - can we get rid of this?

**Review by damccorm — COMMENTED** [review]
Just had 2 minor comments, otherwise LGTM.

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**